imgui/core.pyx

```diff
@@ -1374,6 +1374,68 @@ def is_window_collapsed():
     return cimgui.IsWindowCollapsed()
 
 
+def set_tooltip(str text):
+    """Set tooltip under mouse-cursor.
+    Usually used with :func:`is_item_hovered()`.
+    For a complex tooltip window see :func:`begin_tooltip()`.
+
+    .. visual-example::
+        :auto_layout:
+        :height: 100
+        :width: 200
+        :click: 10 10
+
+
+        imgui.begin("Example: tooltip")
+        imgui.button("Click me!")
+        if imgui.is_item_hovered():
+            imgui.tooltip("Please?")
+        imgui.end()
+
+    .. wraps::
+        void SetTooltip(const char* fmt, ...)
+    """
+    # note: "%s" required for safety and to favor of Python string formating
+    cimgui.SetTooltip("%s", _bytes(text))
+
+
+def begin_tooltip():
+    """Use to create full-featured tooltip windows that aren't just text.
+
+    .. visual-example::
+        :auto_layout:
+        :height: 100
+        :width: 200
+        :click: 10 10
+
+
+        imgui.begin("Example: tooltip")
+        imgui.button("Click me!")
+        if imgui.is_item_hovered():
+            imgui.begin_tooltip()
+            imgui.text("This button is clickable.")
+            imgui.text("This button has full window tooltip.")
+            imgui.image(texture_id, 512, 64, border_color=(1, 0, 0, 1))
```

Comment:
This will not be rendered by our off-screen rendered because `texture_id` is not available in the local scope. You can get some example image using:

    texture_id = imgui.get_io().fonts.texture_id

See: `image()` function docstring